tools/project_manager/diff.php

```diff
@@ -74,13 +74,19 @@
     $format = ($L_format xor $R_format) ? "remove" : "keep";
 }
 
-$query = "
-    SELECT $L_text_column_name, $R_text_column_name,
-           $L_user_column_name, $R_user_column_name
-    FROM $projectid
-    WHERE image='$image'";
-
-$res = mysqli_query(DPDatabase::get_connection(), $query) or die(DPDatabase::log_error());
+$query = sprintf("
```

Comment:
We can't escape column or table names. We should keep the embedded column names as-is and validate the projectID before building the query.

Reply:
Ditto below two changes as well.

Reply:
Actually this is true for several other spots too.